n3fit/src/n3fit/vpinterface.py

```diff
@@ -46,94 +47,34 @@
 ]
 
 
-class N3PDF(PDF):
-    """
-    Creates a N3PDF object, extension of the validphys PDF object to perform calculation
-    with a n3fit generated model.
+class N3Stats(MCStats):
+    """The PDFs from n3fit are MC PDFs
+    however, since there is no grid, the CV has to be computed manually"""
 
-    Parameters
-    ----------
-        pdf_models: :py:class:`n3fit.backends.MetaModel` (or list thereof)
-            PDF trained with n3fit, x -> f(x)_{i} where i are the flavours in the evol basis
-        fit_basis: list(dict)
-            basis of the training, used for reporting
-        name: str
-            name of the N3PDF object
-    """
+    def error_members(self):
+        return self.data
 
-    def __init__(self, pdf_models, fit_basis=None, name="n3fit"):
-        self.fit_basis = fit_basis
-        self.basis = check_basis("evolution", EVOL_LIST)["basis"]
+    def central_value(self):
+        return np.mean(self.data, axis=0)
```

Comment:
Also the same as the vase class.

Reply:
When all members are returned this is no longer true.